external-game-data.js

```diff
@@ -43,31 +43,31 @@
 (function(globals) {
   'use strict';
 
-  var isFirstLoad = true;
+  var ERR_MISSING_IMPORT = 1;
 
-  var _load_game = load_game;
-  globals.load_game = function(game_data, startWithTitle) {
-    // Bitsy caches the game data on first load & recycles it on soft restarts.
-    if (game_data && !isFirstLoad) {
-      return _load_game.apply(this, arguments);
-    }
+  hook('startExportedGame', function beforeStart(superFn, superArgs) {
+    var gameDataElem = document.scripts.namedItem('exportedGameData');
```

Comment:
should just use `document.getElementById` for this

Reply:
You're right; I had a misconception about what's returned from the `scripts` collection vs. document get/query functions.